src/v/serde/serde.h

```diff
@@ -634,6 +642,8 @@ void read_nested(iobuf_parser& in, T& t, std::size_t const bytes_left_limit) {
           read_nested<serde_size_t>(in, bytes_left_limit));
         in.consume_to(str.size(), str.begin());
         t = str;
+    } else if constexpr (std::is_same_v<Type, uuid_t>) {
+        in.read_bytes_to(uuid_t::length, t.uuid.data);
```

Comment:
do you need to add this `read_bytes_to` helper if you use `in.consume_to(uuid_t::length, t.uuid.begin())`, similar to right about this where we do the same thing for strings (`str.begin()`) -- uuid has uuid::begin which returns iterator to underlying data.

Reply:
Done

Reply:
Doh, not sure how I missed that one... Thanks!